lib/create-organization-api.js

```diff
@@ -76,9 +75,7 @@ export default function createOrganizationApi ({ http }) {
    * .catch(console.error)
    */
   function getOrganizationMemberships (query = {}) {
-    return http.get('organization_memberships', {
-      query
-    })
+    return http.get('organization_memberships', createRequestConfig({query: query}))
```

Comment:
nitpick: we can just use the "property value shorthand" here and on line 114, line 179 (we can slowly try to improve the sdk 😅 )